### DISCUSSION TITLE
Why did Leechblock stop working without the new permissions? (Firefox 123)

### DISCUSSION THREAD

**Discussion by JessFairbairn** [proginosko/LeechBlockNG#417] [Q&A] [answered]
HI, long time Leechblock user on Firefox. I noticed it had stopped working on my new machine (Firefox 123) but carried on working on my work machine with an older version of Firefox. I then noticed the newest version of Leechblock has an "optional permissions" to access all data of specific websites, which fixed the blocking when I activated but only for that site. (I eventually activated the permission for all sites).

If this new permission to reflect a change to Firefox needing more permissions?

**Comment by proginosko** [comment, marked as answer]
I don't know why some users have experienced this issue, but the solution is to make sure "Access your data for all websites" is selected under Permissions for LeechBlock NG.
